libs/core/src/lib/multi-input/multi-input-mobile/dropdown-mobile-configuration.ts

```diff
@@ -0,0 +1,15 @@
+export class DropdownMobileConfiguration {
```

Comment:
I don't understand why it's named `Dropdown`.
Maybe we should rather go with `ControlMobileConfiguration `
We should also probably put this into `utils` since this is going to be used across different controls.